src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -670,9 +695,10 @@ export class EditDataEditor extends BaseEditor {
 
 	public toggleQueryPane(): void {
 		this.editDataInput.queryPaneEnabled = !this.queryPaneEnabled();
-		if (this.queryPaneEnabled()) {
+		if (this.sashVisibilityUpdate) {
```

Comment:
I'm assuming you meant to call the function here instead of checking if it's defined (which it'll always be)